roles/forklift_versions/library/forklift_versions.py

```diff
@@ -24,8 +24,12 @@
     description: scenario version to look up
     required: True
   scenario_os:
-    descrption: scenario OS to look up
+    description: scenario OS to look up
     required: True
+  step:
```

Comment:
bikeshed: `upgrade_step`?

Reply:
Yes, I think that would make more sense.